src/main/java/org/candlepin/resteasy/filter/SecurityHoleAuthorizationFilter.java

```diff
@@ -132,10 +134,14 @@ private String checkCloudAccountOrganization(AnonymousCloudConsumer anonymousClo
                 anonymousCloudConsumer.getCloudOfferingId());
         }
         catch (OrgForCloudAccountNotCreatedYetException e) {
-            throw new TooManyRequestsException(ORG_DOES_NOT_EXIST_AT_ALL);
+            String msg = "Upstream anonymous owner not yet created by adapters";
+            throw new TooManyRequestsException(i18nProvider.get().tr(msg))
```

Comment:
You cannot use `tr()` with a variable; it only works with a hardcoded string

Reply:
Updated

Reply:
As a note for when/if this comes up in the future: if we need to have a string that will eventually be translated as a variable (as was done in the original code here), you can have a string primed for translation via `i18n.marktr(<string>)`. That will allow that specific string to be used with `.tr(...)` later without needing to use a string literal. For example:

```
I18n i18n = this.i18nProvider.get();
String msg = i18n.marktr("Some string we want translated");

...

throw new CandlepinException(i18n.tr(msg));
```

Reply:
That is good to know!